ocaml/xapi/xapi_globs.ml

```diff
@@ -730,7 +731,7 @@ let redo_log_max_startup_time = ref 5.
 (** The delay between each attempt to connect to the block device I/O process *)
 let redo_log_connect_delay = ref 0.1
 
-let gpumon_stop_timeout = ref 10.0
+let gpumon_stop_timeout = ref 60.0
```

Comment:
Commit "CA-209508: Use timeout when attempting to stop gpumon" was already in cream-bugfix branch. I deleted this in PR #195 , please review #195.